src/renderer/Components/CreatingPackage/CreatingOrder/BookmarkToolbar/BookmarkList/BookmarkList.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+
+import usePackageStore from "@renderer/store";
+
+function BookmarkList({ bookmarks, closeBookmarkList }) {
+  const { updateOrder } = usePackageStore();
+
+  const applyBookmark = (index) => {
+    delete bookmarks[index]._id;
+    delete bookmarks[index].updatedAt;
+    delete bookmarks[index].createdAt;
+
+    updateOrder(bookmarks[index]);
+    closeBookmarkList();
```

Comment:
사소하긴 한데 최대한 순수함수 유지를 위해 속성을 삭제하는 대신 구조분해 할당으로 속성들을 할당해준뒤 필요한 부분만 넘기는 형태면 어떠시간요?😄

```suggestion
    const { _id, updateAt, createAt, ...rest } = bookmarks[index];
    
    updateOrder(rest);
    closeBookmarkList();
```

Reply:
오 한층 간결하게 코드를 작성 할 수 있겠네요..! 👍